libpod/image/image.go

```diff
@@ -711,7 +717,7 @@ func (i *Image) History(ctx context.Context) ([]*History, error) {
 			Comment:   oci.History[i].Comment,
 		})
 	}
-
+	defer i.newImageEvent(events.History)
```

Comment:
Do we need to record this? It's like Inspect - it doesn't modify anything

Reply:
Agree no "READING" events